### ISSUE TITLE
Le SGB ne suit pas la méthode de conception proposée dans le cours

### ISSUE THREAD

**Issue by Olon11** [profcfuhrmanets/log210-systeme-gestion-bordereau-node-express-ts#9]
Les modèles ne sont pas des classes qui représentent les classes conceptuelles du MDD de SGB. Ils ressemblent plus à des contrôleurs.

Par exemple `Course` ne contient pas de propriété `id`, `prealable` ou `titre`. Il contient une méthode statique qui retourne tous les cours. C'est plutôt le rôle du contrôleur de faire ça. À cause de ça, les contrôleurs ne font que transmettre l'appel de méthode vers le « modèle », ce qui n'apporte rien.

**Comment by fuhrmanator** [comment]
C'est pas important pour l'instant, surtout que tu spécifie que regarder la conception n'est pas le but #10

**Comment by Olon11** [comment]
Oui, je suis d'accord, mais il faut donner l'exemple aux étudiants. Un étudiant pourrait vouloir s'inspirer de la structure du SGB et alors, son projet serait incorrect.

**Comment by Olon11** [comment]
C'est pire que ce que je pensais. Le routeur donne l'objet `query` au contrôlleur, qui le donne à la classse du domaine. Exactement ce qu'on dit aux étudiants de ne pas faire.

Pourquoi `help wanted`? Je peux m'en charger.

**Comment by Olon11** [comment]
J'ai faire une restructuration pour `insert grade` dans e9a570a
